hermes-common/src/main/java/pl/allegro/tech/hermes/common/schema/AvroCompiledSchemaRepositoryFactory.java

```diff
@@ -40,6 +50,20 @@ public AvroCompiledSchemaRepositoryFactory(RawSchemaClient rawSchemaClient, Conf
             public void onTopicRemoved(Topic topic) {
                 repository.removeFromCache(topic);
             }
+
+            @Override
+            public void onTopicCreated(Topic topic) {
+                logger.info("Loading latest schema for created topic {}", topic.getQualifiedName());
+                schemaVersionsRepository.onlineLatestSchemaVersion(topic).ifPresent(
```

Comment:
Can we check whether the topic is avro-type? we'll get rid of unnecessary schema registry requests for json-topics. 
Same on update below.

Reply:
Good idea, done